libcloud/compute/drivers/runabove.py

```diff
@@ -52,13 +39,20 @@ class RunAboveNodeDriver(NodeDriver):
     api_name = 'runabove'
 
     NODE_STATE_MAP = OpenStackNodeDriver.NODE_STATE_MAP
+    VOLUME_STATE_MAP = OpenStackNodeDriver.VOLUME_STATE_MAP
```

Comment:
Yeah, I assumed something like that was the case, but I just wanted to confirm it.

"simple" vs "standard" could be debated, but that's unrelated to this pr :)